src/machines/modelingMachine.ts

```diff
@@ -940,8 +941,7 @@ export const modelingMachine = setup({
           if (!args) return
           if (args.mouseEvent.which !== 1) return
           const { intersectionPoint } = args
-          if (!intersectionPoint?.twoD || !sketchDetails?.sketchEntryNodePath)
-            return
+          if (!intersectionPoint?.twoD) return
```

Comment:
There's a few of these where sketchEntryNodePath was checked for no reason. I think was probably a copy paste thing as to why there are so many of them.